client/functions.lua

```diff
@@ -208,7 +216,7 @@ function QBCore.Functions.GetClosestObject()
             closestDistance = distance
         end
     end
-    return closestObject, closestDistance
+    return closestObject,closestDistance
```

Comment:
Why randomly remove a white space :(

Reply:
Did it work for some reason, it looked bad to me :D